### DISCUSSION TITLE
How to know what flyway version to use

### DISCUSSION THREAD

**Discussion by alexandernajafi** [dropwizard/dropwizard-flyway#554] [Q&A]
The docs cleary states that if I am using Mysql, I also need to add something like 
```
    implementation 'org.flywaydb:flyway-mysql:$flyway.version'
```
But how do I know what `flyway.version` to use? I can see that version `3.0.0-4` (for Dropwizard 3) of this module is using flyway version `9.22.3`, but how can I keep these versions in sync as I upgrade this module? Do I always have to check it manually or can it be linked somehow? I am using dependabot to keep track of updates.

**Comment by joschi** [comment]
@alexandernajafi As of now, you'd have to check the version of Flyway used in this Dropwizard module.

The Flyway BOM is unfortunately still blocked: https://github.com/flyway/flyway/issues/3628
